mathesar/models/query.py

```diff
@@ -446,5 +446,4 @@ def _get_default_display_name_for_group_output_alias(
         .map_of_output_alias_to_input_alias[output_alias]
     input_alias_display_name = current_display_names.get(input_alias)
     if input_alias_display_name:
```

Comment:
@ishir21 This if statement is unnecessary and can be removed, you can directly return `input_alias_display_name`.